docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst

```diff
@@ -0,0 +1,71 @@
+.. _arch_overview_load_balancing_slow_start:
+
+Slow start mode
+===============
+
+Slow start mode is a configuration setting in Envoy to progressively increase amount of traffic for newly added upstream endpoints.
+With no slow start enabled Envoy would send a proportional amount of traffic to new upstream ednpoints.
+This could be undesirable for services that require warm up time to serve full production load and could result in request timeouts, loss of data and deteriorated user experience.
+
+Slow start mode is a mechanism that affects load balancing weight of upstream endpoints and can be configured per upstream cluster. 
+Currently, slow start is supported in Round Robin and Least Request load balancer types.
+
+Users can specify a :ref:`slow start window parameter<envoy_v3_api_field_config.cluster.v3.Cluster.CommonLbConfig.SlowStartConfig.slow_start_window>` (in seconds), so that if endpoint “cluster membership duration" (amount of time since it has joined the cluster) is within the configured window, it enters slow start mode, given that the endpoint satisfies endpoint warming policy. 
+During slow start window, load balancing weight of a particular endpoint will be scaled with :ref:`time bias parameter<envoy_v3_api_field_config.cluster.v3.Cluster.CommonLbConfig.SlowStartConfig.time_bias>`, e.g.:
+`weight = load_balancing_weight * time_bias`.
+
+Whenever a slow start window duration elapses, upstream endpoint exits slow start mode and gets regular amount of traffic acccording to load balanacing algorithm.
+Endpoint could also exit slow start mode in case it leaves the cluster.
+
+To reiterate, endpoint enters slow start mode when:
+  * For :ref:`NO_WAIT<envoy_v3_api_enum_value_config.cluster.v3.Cluster.CommonLbConfig.SlowStartConfig.EndpointWarmingPolicy.NO_WAIT>` endpoint warming policy, immediately if it's cluster membership duration is within slow start window.
+  * For :ref:`WAIT_FOR_FIRST_PASSING_HC<envoy_v3_api_enum_value_config.cluster.v3.Cluster.CommonLbConfig.SlowStartConfig.EndpointWarmingPolicy.WAIT_FOR_FIRST_PASSING_HC>` endpoint warming policy, if it's cluster membership duration is within slow start window and endpoint has passed an active healthcheck. 
+    If endpoint does not pass an active healcheck during entire slow start window (since it has been added to upstream cluster), then it never enter slow start mode.
```

Comment:
I still don't quite understand why we need this configuration. Can we just say that if active HC is configured slow start starts from the time of first passing HC, otherwise it starts immediately? It seems like we don't need this config immediately?